doc/security.md

```diff
@@ -2,7 +2,7 @@
 
 ## Logging in
 
-The simplest way to log into PostgreSQL is by specifying a `Username` and a `Password` in your connection string. Depending on how your PostgreSQL is configured (in the `pg_hba.conf` file), Npgsql will send the password in MD5 or in cleartext (not recommended).
+The simplest way to log into PostgreSQL is by specifying a `Username` and a `Password` in your connection string. Depending on how your PostgreSQL is configured (in the `pg_hba.conf` file), Npgsql will send the password in MD5 or in cleartext (not recommended). By default, Npgsql will look for [PgPassFile](https://www.postgresql.org/docs/current/static/libpq-pgpass.html) if password is not specified. Specify a `PgPassFile` to override the default path.
```

Comment:
It must be clear that the PgPassFile mechanism kicks in only if a Password isn't supplied on the connection string (which is still the standard way to connect).

Also, we need to describe the precise PgPassFile selection logic: first the new connection string parameter you're introducing, then the PGPASSFILE environment variable, and finally the system-dependent default (`$HOME/.pgpass` for Unix, `%APPDATA%\postgresql\pgpass.conf` for Windows).

I can do a pass to amend documentation at the end.